### PR TITLE
Product page

### PR DESCRIPTION
**Pull request by kavyakunder**
product page 

https://user-images.githubusercontent.com/81958043/154570625-3582ac88-e61f-41f1-8a5e-732d9c81ed86.mp4

### CODE REVIEW

File: css/product-page.css [hunk 1]
```diff
@@ -0,0 +1,193 @@
...
+}
+.side-category {
+  height: 90vh;
+  font-size: 1.2rem;
+  background-color: var(--lightest-grey);
+  margin: 4rem 1rem;
+  border-radius: 10px;
+}
+.cat-margin {
+  margin: 40px;
```
Comment (Shivanshsharma01) [inline css/product-page.css]: can use rem here

File: css/product-page.css [hunk 2]
```diff
@@ -0,0 +1,193 @@
...
+  border: none;
+  font-size: large;
+}
+
+.container {
+  position: relative;
+  display: grid;
+  grid-template-columns: repeat(auto-fit, minmax(250px, 1fr));
+  grid-gap: 18px;
+  padding: 20px;
```
Comment (Shivanshsharma01) [inline css/product-page.css]: above 16px try using rem

**Review by Shivanshsharma01 — COMMENTED** [review]
The code looks good and well structured. Suggestion- Try using rem above 16px, I have pointed a few places you can rectify left ones too

**Review by RakshithBhat03 — COMMENTED** [review]
The code looks clean and has good use of variables. Please implement the changes suggested by the other reviewer.

**Pull request merged**